connection_pool/connection_pool_test.go

```diff
@@ -617,8 +617,8 @@ func TestUpdateInstancesRoles(t *testing.T) {
 	args = test_helpers.ListenOnInstanceArgs{
 		ServersNumber: serversNumber,
 		ExpectedPorts: masterPorts,
-		ConnPool: connPool,
-		Mode: connection_pool.PreferRW,
+		ConnPool:      connPool,
```

Comment:
Is this a gofmt issue? I don't remember this change sin previous versions of PR.

Reply:
Yes, this was a gofmt issue. I have updated the commit and PR message.